torch/distributed/algorithms/ddp_comm_hooks/__init__.py

```diff
@@ -38,6 +40,8 @@ def _powerSGD_comm_hook_wrapper(
 
 class DDPCommHookType(Enum):
     """
+    Enumerate ``ddp_comm_hooks`` and ``ddp_comm_hook_wrapper`` partials.
```

Comment:
nit: I think mentioning that they are `functool.partial`s is leaking the implementation details. Maybe it might be better to just say "Enumerate ... communication hook types".